src/components/BrosweArticles.jsx

```diff
@@ -18,13 +18,16 @@ export default function BrowseArticles() {
       {articles.map((article) => {
         return (
           <div className="flex-wrap row-span-2 container mx-auto py-2 box-border max-w-2xl box-decoration-slice bg-stone-200 border-4  border-red-500 shadow-2xl">
-            <p className="text-xl text-center underline">{article.title}</p>{" "}
+            <a className="text-xl text-center underline" href={`http://localhost:5173/article/${article.article_id}`}>{article.title}</a>{" "}
```

Comment:
This link breaks your app. First of all, it won't work in a production environment when you host this later. Secondly, pots can change so localhost may not be able to use 5173 every time. Finally, you have React Router in use so you can use its features to create the correct link here. Think about the route you made. Where should this link lead?